doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -44,3 +46,58 @@ The following table illustrates ABI compatibility between different versions of
 
 ABI compatibility ensures that the Secure Domain and System Controller firmware binaries do not need to be recompiled each time the Application, Radio binaries, or both are recompiled, as long as they are based on a compatible |NCS| version.
 Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binaries components to work together without recompilation when updating to newer |NCS| versions.
+
+nRF54H20 SoC Binaries v0.8.0 changelog
+**************************************
+
+The following sections provide detailed lists of changes by component.
+
+Secure Domain Firmware (SDFW) v.9.0.0
+=====================================
+
+Added
+-----
+
+* Purge protection can be enabled over ADAC.
+* Clock control is enabled in SDFW.
+* Global domain power request service is integrated in SDFW.
+* PUF values from SDROM are cleared on boot.
+
+Updated
+-------
+
+* Local domain reset will trigger a global reset.
+  ``RESETINFO`` will contain both the global and local reset reason.
+* All processors are booted regardless of whether they have firmware.
+  They are booted in halted mode if no firmware is present.
+
+Removed
+-------
+
+* Several services from SSF over ADAC.
+* Reset event service.
+
+Fixed
+-----
+
+* An issue with invoking crypto service from multiple threads or clients.
+
+System Controller Firmware (SCFW) v.4.0.0
+=========================================
+
+Added
+-----
+
+* GDFS service: New service implementation to handle change of global domain frequency on demand (HSFLL120).
+* GDPWR service: new power domains.
```

Comment:
```suggestion
* GDPWR service: New power domains.
```

Reply:
Shouldn't it stay capitalized after colons?